crates/brix_config_loader/src/lib.rs

```diff
@@ -1,9 +1,191 @@
-mod command;
```

Comment:
this file is borderline too big, my limit on line count for file size is that they should generally be no more than 200 lines, 250 max. Exceptions may apply.

Reply:
Although the file is 192 lines I will try to break it up.